release/build_nightly.sh

```diff
@@ -7,6 +8,13 @@ function main {
 
 		LIFERAY_COMMON_DOWNLOAD_SKIP_CACHE="true" LIFERAY_RELEASE_GIT_REF="master" LIFERAY_RELEASE_OUTPUT="nightly" LIFERAY_RELEASE_UPLOAD="true" ./build_release.sh
 
+		if [ "${?}" -ne 0 ]
+		then
+			lc_log ERROR "Nightly build failed."
+
+			exit "${LIFERAY_COMMON_EXIT_CODE_BAD}"
```

Comment:
@kiwm, do we really need it?

I'm asking because I'm concerned about the scenario in which BChan's executes it on his end, e.g. when the `sleep 1d` will be invoked. In this scenario, the `build_nightly.sh` won't retry to build nightly anymore.